isa/kernel.org/instruction-set.rst

```diff
@@ -1,8 +1,22 @@
+.. contents::
+.. sectnum::
 
 ====================
 eBPF Instruction Set
 ====================
 
+The eBPF instruction set consists of eleven 64 bit registers, a program counter,
+and 512 bytes of stack space.
+
+Versions
+========
+
+The current Instruction Set Architecture (ISA) version, sometimes referred to in other documents
+as a "CPU" version, is 3.  This document also covers older versions of the ISA.
+
+*Clang implementation note*: Clang can select the eBPF ISA version using
+``-mcpu=v2`` for example to select version 2.
```

Comment:
Example usage for the `note` directive. But I'm not sure GitHub renders it correctly.

```suggestion
.. note::

    *Clang implementation*: Clang can select the eBPF ISA version using
    ``-mcpu=v2`` for example to select version 2.
```

Reply:
Thanks, but GitHub does not render it correctly, as can be seen in my test at https://github.com/dthaler/ebpf-docs/blob/test-formatting/test/test-formatting.rst#notes

As such, we should not use that syntax in these files.  (I want correct display of a github mirror once merged into kernel.org, like we have mirrors for libbpf and bpftool.)

Reply:
> I want correct display of a github mirror

I didn't know that, understood.

Reply:
There seems to be a way to have notes on GH, although not relying on the traditional RST admonitions, but instead using block quotes: https://github.com/orgs/community/discussions/16925#discussioncomment-3279594

> **Note**
> Haven't tried with RST